### PR TITLE
[CDAP-7352][CDAP-8668] Log viewer bugfixes

### PR DESCRIPTION
**Pull request by tonybach**
[CDAP-7352] https://issues.cask.co/browse/CDAP-7352
- Made headers in the log table sticky (i.e. visible when the user scrolls down)

[CDAP-8668] https://issues.cask.co/browse/CDAP-8668
- Fixed to show ERROR, WARN, and INFO level logs by default, instead of logs from all levels.

EDIT: Was put On-hold because the headers were not sticky yet for pipeline logs and logs in tracker. This has been fixed.

### CODE REVIEW

File: cdap-ui/app/directives/log-viewer/log-viewer.js [hunk 1]
```diff
@@ -111,8 +111,8 @@ function LogViewerController ($scope, LogViewerStore, myLogsApi, LOGVIEWERSTORE_
     'ERROR' : true,
     'WARN' : true,
     'INFO' : true,
-    'DEBUG' : true,
-    'TRACE' : true
+    'DEBUG' : false,
+    'TRACE' : false
   };
 
   vm.selectedLogLevel = 'TRACE';
```
Comment (ajainarayanan) [inline cdap-ui/app/directives/log-viewer/log-viewer.js]: Will this affect the loglevel we set by defaul?

**Comment by ajainarayanan** [comment]
screen shot 2017-03-09 at 2 23 59 pm

I think we have double border on `td`s and `th`s .

File: cdap-ui/app/directives/log-viewer/log-viewer.js (hunk 1 above)
Comment (elfenheart) [reply cdap-ui/app/directives/log-viewer/log-viewer.js]: @tonybach I believe you should set the log level to `INFO`. Check line 613, `includeEvent()`

File: cdap-ui/app/directives/log-viewer/log-viewer.js [hunk 2]
```diff
@@ -488,6 +493,8 @@ function LogViewerController ($scope, LogViewerStore, myLogsApi, LOGVIEWERSTORE_
 
     //Scroll table to the top
     angular.element(document.getElementsByClassName('logs-table'))[0].scrollTop = 0;
+    // binds window element to check whether scrollbar has appeared on resize event
+    angular.element($window).bind('resize', checkForScrollbar);
```
Comment (elfenheart) [inline cdap-ui/app/directives/log-viewer/log-viewer.js]: I believe you need to unbind this on `$destroy`

**Comment by elfenheart** [comment]
Alignment of the header border and table body border is still not quite right

**Comment by leacd** [comment]
It looks good. Merge.

**Comment by elfenheart** [comment]
👍  LGTM

**Comment by tonybach** [comment]
Build: http://builds.cask.co/browse/CDAP-DRC6048

**Pull request merged**